app/controllers/users/reset_passwords_controller.rb

```diff
@@ -79,7 +79,9 @@ def handle_valid_email
     end
 
     def create_account_if_email_not_found
-      user, result = RequestPasswordReset.new(email, request_id).perform
+      user, result = RequestPasswordReset.new(email: email,
+                                              request_id: request_id,
+                                              analytics: analytics).perform
```

Comment:
Definitely not a blocker, but I'd love to have a bikeshed discussion about the two indentation styles in this file and the one above:

```rb
      @register_user_email_form = RegisterUserEmailForm.new(
        recaptcha_results: validate_recaptcha,
        analytics: analytics,
      )
```

```rb
      user, result = RequestPasswordReset.new(email: email,
                                              request_id: request_id,
                                              analytics: analytics).perform
```

...and whether we have a preference to one or the other.

Reply:
I usually do the former, and assumed rubocop would fix the latter. Changed to be consistent with the first option in c4bcc70e3

I'm also happy to bikeshed. The closest checks in Rubocop I could find are:

- https://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Layout/ArgumentAlignment
- https://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Layout/MultilineMethodCallBraceLayout
- https://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Layout/MultilineMethodArgumentLineBreaks

It doesn't look like there are any options to require the first style above. The `MultilineMethodCallBraceLayout` has a `new_line` option that gets close, but isn't quite it.

Reply:
Yeah, my preference would also be to the first, since I find it's much easier to implement and maintain. But ultimately I care less which style it is, and more that there _is_ a preferred style I can align to 😄  

In addition to the cops you found, this one might be relevant as well:

https://docs.rubocop.org/rubocop/cops_layout.html#layoutfirstmethodargumentlinebreak

This mix of configuration seems to get the desired result in my initial testing:

```yml
Layout/ArgumentAlignment:
  Enabled: true
  EnforcedStyle: with_fixed_indentation

Layout/MultilineMethodCallBraceLayout:
  Enabled: true

Layout/FirstMethodArgumentLineBreak:
  Enabled: true
```

Reply:
ah nice, that does it. Running with those options lists 558 issues, with most in the specs. Not sure on the best strategy for fixing that 😬

Reply:
If it's auto-correctable, maybe we can do a quick mega-PR to fix them all at once?

Reply:
> I pushed all of the changes to a branch here: https://github.com/18F/identity-idp/compare/rubocop-multiline-method-calls?expand=1. I'd be happy to pair on this for a bit this week to see if we can iron out those with some option changes.

Yeah, I'd like that 👍  I created some time tomorrow to work through this.

Reply:
Big fan of the new approach (except that weird ending paren all the way on the left site)